### DISCUSSION TITLE
Nuxt support

### DISCUSSION THREAD

**Discussion by martinszeltins** [inertiajs/inertia#1879] [Ideas]
Does Inertia support Nuxt or can it be made to work with Nuxt?

**Comment by ramonmalcolm10** [comment]
Inertia basically provides what the meta-framework does. No need to use Nuxt, Next and or Sveltekit when using Inertia.

**Reply by martinszeltins** [reply]
Does Inertia have modules like Nuxt and auto imports and file based routing and hybrid rendering and suspense and lazy loading and devtools etc.? All these amazing features, it would be strange to miss out on them.

**Reply by ramonmalcolm10** [reply]
I don’t believe it offer any of Nuxt features. It offer more of a glue between Laravel server side and the client side whether you using Vue, React or Svelte. It handle stuff like SPA routing, form submission, data validation and data fetching.

**Reply by martinszeltins** [reply]
True, but it also misses out on all the cool DX features offered by Nuxt :(

**Reply by ramonmalcolm10** [reply]
It’s always a trade off, you will have to determine what trade off work for you
